arangod/Replication2/ReplicatedLog/LogFollower.cpp

```diff
@@ -319,6 +319,18 @@ auto replicated_log::LogFollower::getStatus() const -> LogStatus {
   });
 }
 
+auto replicated_log::LogFollower::getQuickStatus() const -> QuickLogStatus {
+  return _guardedFollowerData.doUnderLock([this](auto const& followerData) {
+    if (followerData._logCore == nullptr) {
+      THROW_ARANGO_EXCEPTION(
+          TRI_ERROR_REPLICATION_REPLICATED_LOG_FOLLOWER_RESIGNED);
+    }
+    return QuickLogStatus{.role = ParticipantRole::kFollower,
+                          .term = _currentTerm,
+                          .local = followerData.getLocalStatistics()};
```

Comment:
shall we add leader established here as well? It would be set to true if the commit index is non-zero.

Reply:
That used to be part only of the `LeaderStatus` before, that's why I have ignored if for followers. Since it's going to take up that space in memory anyway, it would be better indeed to put some proper value in it.